PowerEditor/src/MISC/PluginsManager/Notepad_plus_msgs.h

```diff
@@ -35,7 +35,7 @@
 enum LangType {L_TEXT, L_PHP , L_C, L_CPP, L_CS, L_OBJC, L_JAVA, L_RC,\
 			   L_HTML, L_XML, L_MAKEFILE, L_PASCAL, L_BATCH, L_INI, L_ASCII, L_USER,\
 			   L_ASP, L_SQL, L_VB, L_JS, L_CSS, L_PERL, L_PYTHON, L_LUA,\
-			   L_TEX, L_FORTRAN, L_BASH, L_FLASH, L_NSIS, L_TCL, L_LISP, L_SCHEME,\
+			   L_TEX, L_FORTRAN, L_FORTRAN_77, L_BASH, L_FLASH, L_NSIS, L_TCL, L_LISP, L_SCHEME,\

```

Comment:
I have commit new changes.

in PowerEditor\src\MISC\PluginsManager\Notepad_plus_msgs.h
   PowerEditor\src\ScitillaComponent\ScintillaEditView.cpp

L_COFFEESCRIPT, L_FORTRAN_77,\
instead of
L_TEX, L_FORTRAN, L_FORTRAN_77, L_BASH, L_FLASH, L_NSIS, L_TCL, L_LISP, L_SCHEME,\